### ISSUE TITLE
Zone Filter - Blacklist with incorrect or missing entries

### ISSUE THREAD

**Issue by SayuShira** [MidoriKami/KamiLib#5]
Hey!

I've been trying to write a plugin that only works in specific zones.
To that end, I used your Blacklists e.g. in Chillframe or No Tank You to get the area codes.

Tho, my plugin didn't work in (The) Porta Decumana (the MSQ Trial Instance). Neither does Chillframes stop limiting frames.
![image](https://github.com/MidoriKami/KamiLib/assets/104025906/ad2b72e6-a169-42be-aba6-8340223670aa)

I wanted to dig a bit- and so messed around with Lumina for the first time:
```csharp
        var terri = DataManager.GetExcelSheet<TerritoryType>(ClientLanguage.English);
        var names = DataManager.GetExcelSheet<PlaceName>(ClientLanguage.English);
        // Right for the trial instance: 1048; "The Porta Decumana" 1052; "Porta Decumana": 348
        var porta = terri!.GetRow(348); 
        PluginLog.Information($"Territory: {porta!.PlaceName.Row}");
        
        var name = names!.GetRow(porta.PlaceName.Row);
        PluginLog.Information($"Territory: {name!.Name}");
```

At the moment I don't understand most of that tho. Lumina seems to be even harder to start with than Dalamud q.q
I'm unsure if this is the only affected area/instance.
And was unable to figure out how or where exactly the data was coming together in BlacklistAttribute/LuminaCache, if even there.

---

**tl;dr**: The Blacklist doesn't work on at least Porta Decumana, which should have the number 1048 associated with it (according to my testing).

**Comment by MidoriKami** [comment]
![image](https://github.com/MidoriKami/KamiLib/assets/9083275/c68f00dc-ea9e-4ad1-909f-a93930437802)

The correct one is 1048, my code returns the first result from the name match which is 348, but it seems that value is now unused even though it's still in the gamedata.

**Comment by SayuShira** [comment]
Ahh, I understand.
Thank you for providing the insight. I will look it over with that in mind again.

Otherwise, my goal was to make you aware of this. Which has been accomplished.
So please feel free to close the issue once you see fit.

**Comment by MidoriKami** [comment]
I haven't solved the problem yet. I'm still thinking on a elegant solution.

**Comment by MidoriKami** [comment]
![image](https://github.com/MidoriKami/KamiLib/assets/9083275/2a2e2f64-e4fb-4de9-9a1b-a7174787d5db)
![image](https://github.com/MidoriKami/KamiLib/assets/9083275/1edb0d92-e5d2-4ece-880f-f9be48f859fa)

That should do the trick.
